base-application.gradle

```diff
@@ -6,6 +6,16 @@ android {
     compileSdkVersion compileVersion
     buildToolsVersion buildToolsVersion
 
+    flavorDimensions "extraAPIs"
+    productFlavors {
+        defaults {
+            dimension "extraAPIs"
+        }
+        integrationTest {
+            dimension "extraAPIs"
+        }
```

Comment:
Suggestions for the names of all these are welcome! The main difference is that the `integrationTest` flavor exposes some new APIs, so I thought naming the dimension `extraAPIs` made sense